appeals/web/src/server/lib/dates.js

```diff
@@ -60,6 +60,25 @@ export const dayMonthYearToApiDateString = (dayMonthYear) => {
 	return `${dayMonthYear.year}-${monthString}-${dayString}`;
 };
 
+/**
+ * @param {string | undefined} hour
+ * @param {string | undefined} minute
+ * @returns {string} - time in format HH:MM
+ */
+export const hourMinuteToApiDateString = (hour, minute) => {
+	if (!hour) {
+		return '';
+	}
+	if (!minute) {
+		minute = '0';
+	}
+
+	const hourString = Number(hour) < 10 ? `0${hour}` : hour;
```

Comment:
`.padStart` would be clearer here:

`const hourString = hour.padStart(2, '0');`